### PR TITLE
[SRVCOM-2222] Add Serverless 1.28.0 release notes, update feature tables

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
4.8+

Issue:
https://issues.redhat.com/browse/SRVCOM-2222

Link to docs preview:
* GA/TP table, Deprecated & removed table, and the release notes for the current release: https://58140--docspreview.netlify.app/openshift-enterprise/latest/serverless/serverless-release-notes.html#serverless-rn-1-28-0_serverless-release-notes
* Also, Tech Preview notes for Node.js and TypeScript have been removed in the main docs

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/serverless-rn-1-28-0.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* {ServerlessProductName} now uses Kourier 1.7.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.7.
+* {ServerlessProductName} now uses Knative Kafka 1.7.
+* The `kn func` CLI plug-in now uses `func` FIXME version.
+
+* Node.js and TypeScript runtimes for {ServerlessProductName} Functions are now Generally Available (GA).
+
+* Multi-container support for Knative Serving is now available as a Technology Preview. This feature allows you to use a single Knative service to deploy a multi-container pod.
+
+* In a future release, the following components of Knative Eventing will be scaled down from two pods to one:
```
Comment (mgencur) [inline modules/serverless-rn-1-28-0.adoc:25]: By "future release" you mean 1.29? I'm curious if this is just a warning for the future release.

File: modules/serverless-rn-1-28-0.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* `imc-controller`
+* `imc-dispatcher`
+* `mt-broker-controller`
+* `mt-broker-filter`
+* `mt-broker-ingress`
+--
+
+* The `serverless.openshift.io/enable-secret-informer-filtering` annotation for the Serving CR is now deprecated. The annotation is valid only for Istio, and not for Kourier.
++
+With {ServerlessProductName} 1.28, the {ServerlessOperatorName} allows injecting the environment variable `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID` for both `net-istio` and `net-kourier`.
```
Comment (maschmid) [inline modules/serverless-rn-1-28-0.adoc:37]: Do we document anywhere how to set the ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID env ?  I believe this should be documented as part of https://issues.redhat.com/browse/SRVKS-947 

It seems to me we're now deprecating the annotation but we don't really provide the alternative, as we don't document how to actually do it.

File: modules/serverless-rn-1-28-0.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+[id="new-features-1-28-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.7.
+* {ServerlessProductName} now uses Knative Eventing 1.7.
+* {ServerlessProductName} now uses Kourier 1.7.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.7.
+* {ServerlessProductName} now uses Knative Kafka 1.7.
+* The `kn func` CLI plug-in now uses `func` FIXME version.
```
Comment (jrangelramos) [inline modules/serverless-rn-1-28-0.adoc:19]: Func version in this release is 1.9.1

File: modules/serverless-rn-1-28-0.adoc [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+* Node.js and TypeScript runtimes for {ServerlessProductName} Functions are now Generally Available (GA).
+
```
Comment (jrangelramos) [inline modules/serverless-rn-1-28-0.adoc:22]: I think we could mention that Python is now available as Tech Preview for Functions. @nainaz wdyt?

File: modules/serverless-rn-1-28-0.adoc (hunk 1 above)
Comment (rh-max) [reply modules/serverless-rn-1-28-0.adoc:25]: Yeah, it is indeed just a warning for a future release. I've made it more explicit.

File: modules/serverless-rn-1-28-0.adoc (hunk 2 above)
Comment (rh-max) [reply modules/serverless-rn-1-28-0.adoc:37]: Yes, SRVKS-947 is here https://github.com/openshift/openshift-docs/pull/58234

**Review by skrthomas — COMMENTED** [review]
/lgtm

**Comment by skrthomas** [comment]
> @skrthomas: #58140 failed to apply on top of branch "enterprise-4.8":
> 
> ```
> Applying: Add Serverless 1.28.0 release notes, update feature tables
> Applying: Implement feedback on the release notes
> Applying: Add functions to TP/GA table, trim table down to 3 last releases
> Applying: Improve table column widths
> Applying: Update Tekton task links
> Using index info to reconstruct a base tree...
> M	modules/serverless-functions-creating-on-cluster-builds.adoc
> Falling back to patching base and 3-way merge...
> Auto-merging modules/serverless-functions-creating-on-cluster-builds.adoc
> CONFLICT (content): Merge conflict in modules/serverless-functions-creating-on-cluster-builds.adoc
> error: Failed to merge in the changes.
> hint: Use 'git am --show-current-patch=diff' to see the failed patch
> Patch failed at 0005 Update Tekton task links
> When you have resolved this problem, run "git am --continue".
> If you prefer to skip this patch, run "git am --skip" instead.
> To restore the original branch and stop patching, run "git am --abort".
> ```

@rh-max can you create a PR to manually cherrypick this to 4.8? Please let me know when you have the PR ready and I'll merge it. Thanks!

**Comment by skrthomas** [comment]
@rh-max Hey Max, another merge reviewer brought to my attention- I wasn't thinking at the time, 4.8 is not a version we are currently updating. So you will need to reach out to OCP DPM to get the 4.8 update published on customer portal. Technically we are only supposed to merge important/critical items to 4.8.

**Pull request merged**